app.js

```diff
@@ -1,3 +1,5 @@
+global.config = require('./config.json');
```

Comment:
done

On Mon, Aug 3, 2015 at 5:53 PM, Raül Pérez <notifications@github.com> wrote:

> In app.js
> <https://github.com/repejota/nats-mon/pull/11#discussion_r36099172>:
>
> > @@ -1,3 +1,5 @@
> > +global.config = require('./config.json');
>
> var config = require('./config.json'); is enough
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/repejota/nats-mon/pull/11/files#r36099172>.
>



-- 

*Adrià Cidre*
------------------------------------
E-Mail <adria.cidre@gmail.com> - Web Page <http://en.blog.oridoki.com/> -
Linkedin <http://www.linkedin.com/in/adriacidre>
Skype: *kumulo*
Twitter: @adriacidre <http://twitter.com/adriacidre>